app/scripts/controllers/metametrics.js

```diff
@@ -944,10 +955,36 @@ export default class MetaMetricsController {
         return resolve();
       };
 
-      this.segment.track(payload, callback);
+      this._submitSegmentEvent('track', payload, callback);
       if (flushImmediately) {
         this.segment.flush();
       }
     });
   }
+
+  // Method below submits the request to analytics SDK.
+  // For MV3 it will also add event to controller store
```

Comment:
this method appears to add the event to the store whether or not it is MV3. might want to update the code to reflect this comment 

Based on the below "event" -> "APICall" rename suggestion, I'd like to suggest updating the comment to something like
```
  /**
   * Use the method below to submit API calls to Segment using the analytics SDK.
   *
   * For MV3, this method also adds the API call to the controller store and removes
   * the API call from the controller store once its callback is executed.
   * Saving API calls in the controller store enables us to ensure API calls are submitted
   * to Segment even if the service worker termintates before the API call is submitted.
   *
   * @param {string} apiCallMethod - The method name of the analytics API call to be submitted to Segment.
   *                                 e.g. identify, track, page, etc.
   * @param {object} payload
   * @param {Function} callback
   */
```